src/zenml/integrations/aws/orchestrators/sagemaker_orchestrator.py

```diff
@@ -434,3 +480,202 @@ def get_pipeline_run_metadata(
         )
         run_metadata[METADATA_ORCHESTRATOR_URL] = Uri(orchestrator_logs_url)
         return run_metadata
+
+    @staticmethod
+    def _dissect_pipeline_execution_arn(
```

Comment:
I think this is duplicated with the function at the top of this file?

Reply:
Absolutely. I moved the method outside but forgot to remove the one in the class. Removed it now, thanks.